core/controllers/collection_editor_test.py

```diff
@@ -52,21 +49,6 @@ def setUp(self):
             }]
         }
 
-    def assert_can_edit(self, response_body):
-        """Returns True if the response body indicates that the collection is
-        editable.
-        """
-        self.assertIn(self.CAN_EDIT_STR, response_body)
-        self.assertNotIn(self.CANNOT_EDIT_STR, response_body)
-
-    def assert_cannot_edit(self, response_body):
```

Comment:
I think not having helpers is fine, but can you make sure we have tests verifying the rights handler still works in cases where the user does and does not have edit access to a given collection? I think that's what's ultimately missing with the new test (it's only testing one of several permission scenarios for a user).